portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,38 +21,67 @@
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
 import java.util.List;
-import com.google.gson.Gson;
 import java.util.Arrays;
+import java.sql.Timestamp;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
```

Comment:
shall we sort these as well？

Reply:
Sort as in place them alphabetically, right? 
If yes, will do that.